src/eventobserver/EventObserver.ts

```diff
@@ -0,0 +1,13 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import EventAttributes from '../eventcontroller/EventAttributes';
+import EventName from '../eventcontroller/EventName';
+
+export default interface EventObserver {
+  /**
+   * Called when specific events occur during the meeting and includes attributes of the event. This can be used to
+   * create analytics around meeting metric.
+   */
+  eventDidReceive?(name: EventName, attributes: EventAttributes): void;
```

Comment:
Lol, if this is the only function...probably doesn't have to be optional right? @simmkyu

Reply:
Lol yeah you're right, really no need to make it optional.